Source/EasyNetQ/Consumer/Consumer.cs

```diff
@@ -223,12 +224,18 @@ private void RestartConsumingPeriodically()
         if (consumerToRestart == null) return;
 
         var status = consumerToRestart.StartConsuming(false);
+
+        foreach (var queue in status.Started)
+            eventBus.Publish(new StartConsumingSucceededEvent(this, queue));
```

Comment:
@Pliner Maybe rename `StartConsumingSucceededEvent` to `StartConsumingStartedEvent` ?